graylog2-server/src/main/java/org/graylog2/alerts/types/FieldContentValueAlertCondition.java

```diff
@@ -61,6 +64,33 @@ FieldContentValueAlertCondition create(Stream stream,
                                                @Assisted("userid") String creatorUserId,
                                                Map<String, Object> parameters,
                                                @Assisted("title") @Nullable String title);
+
+        Config config();
+        Descriptor descriptor();
+    }
+
+    public static class Config implements AlertCondition.Config {
+        public Config() {
+        }
+
+        @Override
+        public ConfigurationRequest getRequestedConfiguration() {
+            return ConfigurationRequest.createWithFields(
+                new TextField("field", "Field", "", "Field name that should be checked", ConfigurationField.Optional.NOT_OPTIONAL),
+                new TextField("value", "Value", "", "Value that the field should be checked against", ConfigurationField.Optional.NOT_OPTIONAL)
+            ).addFields(AbstractAlertCondition.getDefaultConfigurationFields());
+        }
+    }
+
+    public static class Descriptor extends AlertCondition.Descriptor {
+        public Descriptor() {
+            super(
+                "Field Content Value Alert Condition",
+                "http://www.graylog.rog",
+                "This condition is triggered when the value of a defined field in a defined time interval has a "
+                + "mean/mininum/maximum value/standard deviation which is higher/lower than a defined threshold."
```

Comment:
Maybe "This condition is triggered when the aggregated value of a field is higher/lower than a defined threshold for a given time range."?